x-pack/plugins/uptime/e2e/playwright_start.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable no-console */
+
+import Url from 'url';
+import { run as playwrightRun } from '@elastic/synthetics';
+import { esArchiverLoad, esArchiverUnload } from './tasks/es_archiver';
+
+import './journeys';
+
+export function playwrightRunTests() {
+  return async ({ getService }: any) => {
+    try {
+      const result = await playwrightStart(getService);
+
+      if (result && result.uptime.status !== 'succeeded') {
+        process.exit(1);
+      }
+    } catch (error) {
+      console.error('errors: ', error);
+      process.exit(1);
+    }
+  };
+}
+
+async function playwrightStart(getService: any) {
+  console.log('Loading esArchiver...');
```

Comment:
Is there a logger utility Kibana exposes for us to call instead of bare `console.log`? It looks kinda weird in the output. Not a huge deal, esp. for v1, but if it's easy it could be nicer to improve this.

![image](https://user-images.githubusercontent.com/18429259/134407743-713e8f06-226c-47d8-a167-a5bb29cdbbcd.png)

Reply:
defenitely something worth doing in a follow up, probably we will use junit output to hook the logs into jenkins ci

Reply:
Can we add an enhancement issue then to make sure we don't lose track of it? Should be a fairly light addition someone can implement when they have time.

Reply:
Sure, i will add that.